include/api/CDataFrameAnalysisSpecification.h

```diff
@@ -91,14 +94,17 @@ class API_EXPORT CDataFrameAnalysisSpecification {
     //! \note temp_dir Is a directory which can be used to store the data frame
     //! out-of-core if we can't meet the memory constraint for the analysis without
     //! partitioning.
-    CDataFrameAnalysisSpecification(const std::string& jsonSpecification);
+    //! \param persistStream Shared pointer to the string with  persist stream name.
```

Comment:
This comment is out-of-date now:
```suggestion
    //! \param persistStreamSupplier Provides the stream to which to persist state during the analysis run.
```

Reply:
Thank you for catching this! It's fixed now.